scripts/installUbuntuDeps.sh

```diff
@@ -91,6 +100,12 @@ install_conan(){
   pip3 install conan==1.21
 }
 
+install_cpplint(){
+   sudo pip3 install cpplint==1.5.4
```

Comment:
Yes we should because all scripts no longer call the cpplint.py file that was removed. I don't have access to a Mac so if you could do it it would be perfect.